packages/terra-site/src/App.jsx

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
do we need this file, Index, or TestLinks?

Reply:
These were added by mistake during a rebase. Removed [here](https://github.com/cerner/terra-core/commit/b6399f6704e36e5495678f8832b516bc88328f3e).